### PR TITLE
Header layout reworked (partial)

### PR DESCRIPTION
**Pull request by gagarin55**
- new fonts added
- some colors changed according to new design
- header refactored

### CODE REVIEW

**Comment by splix** [comment]
Instead of putting fonts into sources, can we pull them as a dependency? There are few NPM packages for that, which can bring Google Fonts. I remember I was using one them, don't know which one

File: src/index.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ import {store, start as startStore} from './store/store.js';
 import log from 'electron-log';
 import 'roboto-fontface/css/roboto/sass/roboto-fontface.scss'
 import './index.scss'
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+const muiTheme = getMuiTheme({
+    fontFamily: 'Poppins Regular',
+});
```
Comment (elaineo) [inline src/index.js]: According to the Invision thing, the font is "GT Walsheim Pro". Is "Poppins" close enough? If so, I will update my changes to use your font.

Comment (splix) [inline src/index.js]: Yes, it's close enough. GT Walsheim Pro is paid font, and we can distribute it with open source project

**Comment by gagarin55** [comment]
`typeface-poppins` package has been added

**Comment by splix** [comment]
@gagarin55 you can merge it if it's finished. if github doesn't allow it, let me know

**Comment by gagarin55** [comment]
This is the last commit in this PR ;) I've decided not to use react-flexbox components and refactored layout in plain CSS flexbox, it's more transparent and maintainable.

**Pull request merged**